.github/workflows/pull_request.yml

```
@@ -23,29 +23,21 @@ jobs:
     runs-on: ubuntu-latest
     name: Maven Build
     steps:
-      - name: Setup Maven And Java Version
-        uses: s4u/setup-maven-action@v1.2.1
+      - name: Checkout actions repository
+        uses: actions/checkout@v2
+        with:
+          repository: kiegroup/droolsjbpm-build-bootstrap
+          path: composite-actions
+          ref: tests/composite-actions
```

it will be (just for the projects with the same branch (7.x projects will always point to main branch)
```suggestion
          ref: ${{ github.base_ref }}
```